packages/tokens-studio-for-figma/src/app/store/providers/gitlab/gitlab.tsx

```diff
@@ -101,7 +101,7 @@ export function useGitLab() {
           themes,
           metadata: {},
         };
-      } catch (e) {
+      } catch (e: any) {
```

Comment:
```suggestion
      } catch (e: Error) {
```

This would save checking `e instanceof Error` below, no?

Reply:
No. The error type `Gitbeaker` module returns is not Error.
So it need to be flexible.